bindings/go/examples/get_object.go

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) 2025 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+package main
+
+import (
+	"encoding/hex"
+	"fmt"
+	"log"
+
+	sdk "bindings/iota_sdk_ffi"
+)
+
+func isNilError(err error) bool {
+	if sdkErr, ok := err.(*sdk.SdkFfiError); ok {
+		return sdkErr == nil
+	}
+	return false
+}
+
+func main() {
+	client := sdk.GraphQlClientNewDevnet()
+
+	objectID, err := sdk.ObjectIdFromHex("0x20c056090c3dd1604fcfd7ea759781de650aa45323738e799365d0c28bebeb1e")
+	if err != nil {
+		log.Fatalf("Failed to parse object ID: %v", err)
+	}
+
+	objOpt, err := client.Object(objectID, nil)
+	if !isNilError(err) {
+		log.Fatalf("Failed to get object contents: %v", err)
+	}
+	if objOpt == nil {
+		log.Fatal("Missing object")
+	}
+	obj := *objOpt
+
+	objType := "Package"
+	if obj.ObjectType().IsStruct() { 
+		objType = obj.ObjectType().AsStruct().String()
+	}
+
+	fmt.Println("Object ID:", obj.ObjectId().ToHex())
+    fmt.Println("Version:", obj.Version())
+    fmt.Println("Previous transaction:", obj.PreviousTransaction().ToBase58())
+    fmt.Println("Shared:", obj.Owner().IsShared())
```

Comment:
How about `Owner: Shared` if that's true or `Owner : 0x...` in case of an address owned object?